CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ Guide: https://keepachangelog.com/en/1.0.0/
 
 ## Unreleased
 
+- [Core] Updated to Xcode 14.1 minimum version
+- [Core] Updated deployment target to iOS 14
```

Comment:
Is it really necessary to increase the deployment target to iOS 14? E.g. [Maps SDK uses 12.0](https://github.com/mapbox/mapbox-maps-ios/blob/main/Package.swift#L15), [Navigation v2 SDK uses also 12.0](https://github.com/mapbox/mapbox-navigation-ios/blob/2d425ca123ddeaeff33b5a26cd5c438dacce7b5c/Package.swift#L9).

Btw, the minimum iOS in [Package.swift](https://github.com/mapbox/mapbox-search-ios/blob/bd4b2642d3c4e3de4a864eacc23ed4ad6e59482c/Package.swift#L15) is still iOS 11

Reply:
Thank you! Okay yes that makes more sense to stay consistent with the other SDKs and requiring Xcode 14.